scss/modules/_data-container.scss

```diff
@@ -76,6 +76,54 @@
   padding-top: u(.5rem);
 }
 
+.report-type-col {
+  .document-description {
+    margin-right: 0;
+  }
+
+  .dropdown {
+    margin-top: u(.5rem);
+  }
+
+  .dropdown__panel {
+    min-width: 70px;
+    left: 0;
+    right: auto;
+    margin-top: u(-.5rem);
+  }
+
+  .dropdown__value {
+    padding: u(0 .75rem);
+  }
+}
+
+@include media($med) {
+  .report-type-col {
+    position: relative;
+
+    .document-description {
+      margin-right: u(6.5rem);
+      padding: u(.25rem 0);
+    }
+
+    .dropdown {
+      position: initial;
+    }
+
+    .dropdown__panel {
+      right: u(1rem);
+      left: auto;
+      margin-top: u(.15rem);
```

Comment:
Good catch. I wasn't seeing it because I was looking at it the large breakpoint. Using `top` property values instead for more accuracy.